### ISSUE TITLE
Be able to request access to information when creation is conflicting between organizations - First part

### ISSUE THREAD

**Issue by SamuelHassine** [LuatixHQ/opencti#3345]
## Use case

Scenario:

1. Organization A created IP Address 1
2. Organization B creates IP Address 1
3. Result = Error already exists / restricted

In the "error dialog", be able to click a on link to "request the access" to IP Address 1.

The request should (discussed with @richard-julien):

* Open a RFI case associated to the IP address 1
* Automatically assign the case to ??? (to be discussed)

NEED: Modelize properly to whom we need to share the information contained in the RFI (description ? organization in the case ?)

Then, the team assigned to the case can share subsequent / requested information.

For a better experience, this issue is blocked by #3181 and #3198.

In the case if the owner of the information is the platform organization, no problem.
In the case if the owner of the information is another organization (and the platform organization would like to not validate itself all the requests), this issue is blocked by https://github.com/OpenCTI-Platform/opencti/issues/3346 (be able to share something on purpose when having the capability but not part of the platform organization).
